data_model/primitives/src/fixed.rs

```diff
@@ -7,6 +7,7 @@ use derive_more::Display;
 use fixnum::{
     ops::{CheckedAdd, CheckedSub, Zero},
     ArithmeticError,
+    _priv::Bounded,
```

Comment:
It's available in [ops](https://docs.rs/fixnum/0.6.1/fixnum/ops/trait.Bounded.html), i think

Reply:
Good point